integration/controllers/carpool.ctrl.spec.ts

```diff
@@ -300,6 +314,42 @@ describe('Carpool controller', () => {
       .finally(done);
   });
 
+  it('should update a carpool', (done) => {
+    var carpool:models.Carpool;
+    var carpoolUpdate = {
+      'name': 'name updated',
+      'description': 'description updated'
+    };
+
+    function test(status) {
+      expect(status).toBe(204);
+
+      var updatedCarpool = CarpoolSvc.getCarpoolByID(carpool.id);
```

Comment:
Doe this work? Doesn't `CarpoolSvc.getCarpoolByID` return a `Promise<Carpool>` and not `Carpool`? The `test()` function will fail.